deploy/addons/registry/registry-rc.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    kubernetes.io/minikube-addons: registry
+  name: registry
+  namespace: kube-system
+spec:
+  replicas: 1
+  selector:
+    kubernetes.io/minikube-addons: registry
```

Comment:
These are ran by the addon-manager so they need the appropriate labels for it
Something like:
https://github.com/kubernetes/minikube/blob/master/deploy/addons/kube-dns/kube-dns-controller.yaml#L24